TreeMaker/python/makeTreeFromMiniAOD_cff.py

```diff
@@ -243,19 +242,17 @@ def makeTreeFromMiniAOD(self,process):
     JetAK8TagInf = cms.InputTag('slimmedJetsAK8Inf')
     SubjetTag = cms.InputTag('slimmedJetsAK8PFPuppiSoftDropPacked:SubJets')
 
-    process.load("CondCore.DBCommon.CondDBCommon_cfi")
-    from CondCore.DBCommon.CondDBSetup_cfi import CondDBSetup
+    process.load("CondCore.CondDB.CondDB_cfi")
     
     # get the JECs (disabled by default)
     # this requires the user to download the .db file from this twiki
     # https://twiki.cern.ch/twiki/bin/viewauth/CMS/JECDataMC
     if len(self.jecfile)>0:
         #get name of JECs without any directories
         JECera = self.jecfile.split('/')[-1]
-        JECPatch = cms.string('sqlite_file:'+self.jecfile+'.db')
+        process.CondDB.connect = cms.string('sqlite_file:'+self.jecfile+'.db')
```

Comment:
I dumped the config after this change to check this, and it seems to work correctly because each actual `ESSource` is created by *cloning* `process.CondDB`.